sdclientapi/__init__.py

```diff
@@ -146,16 +146,18 @@ def authenticate(self, totp: Optional[str] = None) -> None:
             raise BaseError("Error in parsing JSON")
         if "expiration" not in token_data:
             raise AuthError("Authentication error")
-        self.token = token_data["token"]
+        self.token = token_data
```

Comment:
I don't think we should just blindly make this a `Dict[str, str]`. If there's values we want, we should parse them out, and if we want to save the rest that should be saved into a `token_extra: Dict[str, str]`. I feel like this seems more natural from a typechecking point of view.

Reply:
I am okay with both sides, at first I wanted to store the whole token data in case we want to use it later, I don't think we are using it anywhere else later.

Reply:
The client is still expecting the dictionary, and uses the `journalist_uuid` component from it, which isn't present on `API` after authentication as of 0.0.7. I haven't worked on the client or SDK enough to have an opinion; I was just trying to unblock client work. If we decided on explicit fields with type hints, it looks like we just need to make sure we bring `journalist_uuid` along too. 

I'm going to pause and let you guys, @redshiftzero and @creviera sync up here.

Reply:
If we need the UUID, we should parse that into a field on the object and document it. I think we shouldn't be relying on consumers to parse what is raw API data. This is also a better fix for type checking.